src/index.html

```diff
@@ -11,7 +11,178 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+
+        <label class="form-field top">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            id="surname"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="form-field">
+          Name:
+          <input
+            type="text"
+            name="name"
+            id="name"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="form-field">
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            id="age"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+
+        <label class="form-field">
+          Full date of birth:
+          <input
+            type="date"
+            name="dateOfBirth"
+            id="dateOfBirth"
+            placeholder="mm/dd/yyyy"
+          >
+        </label>
+
+        <label class="form-field bot">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="termAgreement"
+            id="termAgreement"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+
+        <label class="form-field top">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            id="email"
+            placeholder="email@example.com"
+            required
+          >
+        </label>
+
+        <label class="form-field bot">
+          Password:
+          <input
+            type="password"
+            name="password"
+            id="password"
+            required
+            minlength="8"
+            maxlength="24"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+
+        <label class="form-field top">
+          Do you love cats?
+          <input
+            type="radio"
+            value="y"
+            name="cat"
+          >
+          Yes
+          <input
+            type="radio"
+            value="n"
+            name="cat"
+          >
```

Comment:
Let's use more meaningful values

```suggestion
          <input
            type="radio"
            value="yes"
            name="cat"
          >
          Yes
          <input
            type="radio"
            value="no"
            name="cat"
          >
```